spec/features/update_username_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Update username controller', type: :feature, js: true do
+  let!(:user) { create(:user, id: 500, username: 'Temp') }
```

Comment:
This user is only used for logging in; there's no need to specify an ID or username, and there's no need to use `let!` instead of `let`, since it gets called in the `before` block.

Reply:
@ragesoss if i dont specify ID and username here, then it was creating default user which was with username 'Ragesock' and same global_id which i need to use in below test case, So, it was giving DiplicateEntry Error

Reply:
You can specify a username without an `id`. There's no need for any explicit `id` values in this test file.